src/custom/components/Tokens/TokensTable.tsx

```diff
@@ -159,15 +176,24 @@ export default function TokenTable({
     }
   }, [maxItems, tokensData])
 
+  // for small screens, auto-scrolls table to the left on the page change
+  useEffect(() => {
+    if (prevPageIndex !== page && tableRef.current) {
+      tableRef.current.scrollLeft = 0
+    }
+  }, [page, prevPageIndex])
```

Comment:
Not sure about this. Probably should disappear if we end up doing a more responsive design in a future PR